scripts/dats_validator/validator.py

```diff
@@ -267,7 +267,10 @@ def validate_is_about(dataset):
     if "isAbout" in dataset.keys():
         species_present = False
         for entry in dataset["isAbout"]:
-            if "identifier" in entry.keys():
+            if (
+                "identifier" in entry.keys()
+                and "identifierSource" in entry["identifier"].keys()
```

Comment:
I know this is what the validator was checking before too, but I think `identifierSource` is not the place where the IRI should go. See https://github.com/CONP-PCNO/schema/issues/8

We may want to discuss this there first.

Reply:
@surchs could you add something to the agenda of the CONP dev call to talk about that? I took the examples we have in all of our DATS files. If this is invalid, that means most DATS files are invalid and the DATS GUI is generating invalid encoding for the species. Worth discussing at the dev call. Thanks!

Reply:
OK, will do.